frontend/src/app/workspace/workspace.component.html

```diff
@@ -3,14 +3,21 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<div class="center">
-  <h1 class="center-text"><b>Request Session</b></h1>
-  <mat-spinner *ngIf="showSpinner"></mat-spinner>
-  <app-no-repository
-    *ngIf="!repositories.length && !showSpinner"
-  ></app-no-repository>
-  <app-request-session
-    *ngIf="repositories.length"
-    [ownProjects]="repositories"
-  ></app-request-session>
-</div>
+<h1>Persistent Workspace</h1>
+
+<p>
+  Persistent Workspaces will store all the changes you make on models.
+  Additionally, newly created projects will be stored in your personal
+  workspace.
```

Comment:
```suggestion
  Additionally, newly created models will be stored in your personal
  workspace.
```

Models should be the better name here, because projects is already used by our application in another context.